libzhlgen/test/functions/win32/1.6.9/HackingSystem.zhl

```diff
@@ -2,3 +2,5 @@
 __thiscall void HackingSystem::BlowHackingDrone(HackingSystem *this);
 "578d7c240883e4f0ff77fc5589e557565389cb83ec4ce8????????80bbe4010000000f84????????80bbc0010000000f84????????8b8bac070000":
 __thiscall void HackingSystem::OnLoop(HackingSystem *this);
+"80b9c00100000075??31c0c38d7426005589e55389cb83e4??e8????????83f8??74??":
+__thiscall bool HackingSystem::SoundLoop(HackingSystem *this);
```

Comment:
Need signatures for Linux before we can merge, are you able to get the Linux binary from Steam?

If you have GoG or Humble Bundle copies it's even easier you can get it directly from their websites.

### Here's the instructions for getting it from Steam:
Also on Steam you can very easily get the Linux version:

You'll need to open the steam console https://www.addictivetips.com/windows-tips/open-steam-console/ then we're following this guide except we're not downloading an older version of a game https://www.reddit.com/r/Steam/comments/611h5e/guide_how_to_download_older_versions_of_a_game_on/

We're interested in the latest Linux depot https://steamdb.info/app/212680/depots/ listed as 221002

So the command should be `download_depot 212680 221002`
Steam should then print out where it put the files

Reply:
I was able to get the Linux binary from Humble Bundle I believe, still need to get the one from Steam. Will work on cracking that open and grabbing the signatures.

Reply:
Added Linux sigs